components/browser/domains/src/main/java/mozilla/components/browser/domains/DomainAutoCompleteProvider.kt

```diff
@@ -44,24 +69,26 @@ class DomainAutoCompleteProvider {
         val searchText = rawText.toLowerCase(Locale.US)
 
         if (useCustomDomains) {
-            val autocomplete = tryToAutocomplete(searchText, customDomains)
-            if (autocomplete != null) {
+            val result = tryToAutocomplete(searchText, customDomains)
+            if (result != null) {
+                val (autocomplete, domain) = result
                 val resultText = getResultText(rawText, autocomplete)
-                resultCallback(resultText, AutocompleteSource.CUSTOM_LIST, customDomains.size)
+                resultCallback(resultText, domain, AutocompleteSource.CUSTOM_LIST, customDomains.size)
```

Comment:
Should we pass a String to the resultCallback instead e.g. domain.toString() which could return the protocol + (www) + host? The advantages is that the caller (Focus) doesn't have to assemble the string.

Reply:
@csadilek yes, change incoming.